readthedocs/projects/models.py

```diff
@@ -141,7 +141,7 @@ class Project(models.Model):
     skip = models.BooleanField(_('Skip'), default=False)
     mirror = models.BooleanField(_('Mirror'), default=False)
     use_virtualenv = models.BooleanField(
-        _('Use virtualenv'),
+        _('Install Project'),
```

Comment:
Please keep in mind that changing any field argument will require adjustments in the migrations:
![screen1](https://cloud.githubusercontent.com/assets/88278/9380192/de5c086a-472e-11e5-9759-94c4ef45b6d1.png)
(See the "Alter field use_virtualenv on project")

I usually go the way to change the verbose_name of the original migration which introduced the field since I'm 100% sure that this change will not affect the DB. The other probably cleaner solution (which is causing other problems when using many long lived feature branches) is to create a new migration that changes the field.